notify/impl.go

```diff
@@ -1285,16 +1285,39 @@ const (
 	victorOpsEventResolve = "RECOVERY"
 )
 
-type victorOpsMessage struct {
-	MessageType       string `json:"message_type"`
-	EntityID          string `json:"entity_id"`
-	EntityDisplayName string `json:"entity_display_name"`
-	StateMessage      string `json:"state_message"`
-	MonitoringTool    string `json:"monitoring_tool"`
-}
-
 // Notify implements the Notifier interface.
 func (n *VictorOps) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
+
+	var err error
+	var (
+		data   = n.tmpl.Data(receiverName(ctx, n.logger), groupLabels(ctx, n.logger), as...)
+		tmpl   = tmplText(n.tmpl, data, &err)
+		apiURL = n.conf.APIURL.Copy()
+	)
+	apiURL.Path += fmt.Sprintf("%s/%s", n.conf.APIKey, tmpl(n.conf.RoutingKey))
+
+	c, err := commoncfg.NewClientFromConfig(*n.conf.HTTPConfig, "victorops")
+	if err != nil {
+		return false, err
+	}
+
+	buf, err := n.createVictorOpsPayload(ctx, as...)
+	if err != nil {
+		return true, err
+	}
+
+	resp, err := post(ctx, c, apiURL.String(), contentTypeJSON, buf)
+	if err != nil {
+		return true, err
+	}
+
+	defer resp.Body.Close()
+
+	return n.retry(resp.StatusCode)
+}
+
+// Create the json payload to be sent to victorops api.
```

Comment:
s/json/JSON/
s/victorops api/the VictorOps API/

Reply:
Thanks, just pushed the fix